BIDLicenses.js

```diff
@@ -57,7 +57,7 @@ const getCurrentLicense = async (licenseKey, serviceUrl, myKeyPair, requestUID =
     const requestId = JSON.stringify({
         ts: Math.round(new Date().getTime() / 1000),
         appid: senderId,
-        uuid: requestUID
+        uuid: requestUID.uuid
```

Comment:
We should copy over all the other attributes of the passed in requireUID rather than just the uuid

Reply:
We should have to connect and discuss this.